### ISSUE TITLE
Improve performance of WET-BOEW

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew#8921]
Ian Ward posted a note in GCMessage with "I’m seeing continuous CPU usage on my site even when it’s a background tab. Seems like something in wet is using setInterval repeatedly even for static pages. I see the same thing on https://wet-boew.github.io/wet-boew/demos/index-en.html is this a known issue?"

Using FF `about:performance` it is possible to gather performance of tabs.  I did this and got a few results here. 

<img width="1343" alt="Screenshot with firefox performance tabs" src="https://user-images.githubusercontent.com/116832/90194967-69517380-dd96-11ea-81bc-a405a16a68df.png">

This shows that there is some load which is hard to explain. 

One of the folks behind the Green Web Foundation pointed me to:
https://profiler.firefox.com/
https://developer.mozilla.org/en-US/docs/Tools/Performance

And also to the issue where he was looking into this in more detail:
https://github.com/Greening-Digital/browser-green-o-tron/issues/2

Static pages shouldn't have much of any load. Certainly there are others that don't have any load. 

Would be good to ensure we're reducing the load for CO2 if not for basic battery life.

**Comment by Pacoup** [comment]
Yeah, I'm seeing the same thing in Chrome, backgrounded tab or not. It's due to the timerpoke mechanism.

![image](https://user-images.githubusercontent.com/112764/90660646-2ff48a00-e214-11ea-8a48-a2553b6b3f82.png)
![image](https://user-images.githubusercontent.com/112764/90660686-3d117900-e214-11ea-8f55-00e0247c3171.png)

Save for rearchitecting the whole thing, perhaps the timer can be paused when the page is backgrounded?
[Page Visibility API - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API)

Some of the features used on a page could be causing unusually expensive calls to timerpoke as well, I think. A user had problems once because of some of my dumb code:
[Remove unnecessary timer event](https://github.com/wet-boew/wet-boew/pull/8256/files)
[Do not block collapsible component init on whole document](https://github.com/wet-boew/wet-boew/pull/8266/files) (timerpoke runs the init if I understand correctly)
